cfgmgr/intfmgr.cpp

```diff
@@ -112,10 +112,21 @@ void IntfMgr::setIntfIp(const string &alias, const string &opCmd,
         (cmd << IP_CMD << " -6 address " << shellquote(opCmd) << " " << shellquote(ipPrefixStr) << " dev " << shellquote(alias) << metric);
     }
 
-    int ret = swss::exec(cmd.str(), res);
-    if (ret)
+    int failed = swss::exec(cmd.str(), res);
+    if (failed && !ipPrefix.isV4() && opCmd == "add")
```

Comment:
Can you please use the previous "ret" variable and add the remaining check within the failed case.

Reply:
Handled